src/modules/commands/fight.js

```diff
@@ -0,0 +1,22 @@
+const isEmpty = require('lodash/isEmpty');
+const random = require('lodash/sample');
+const command = require('../command.filter');
+
+const fights = async function (request, context) {
+    const { i18n, send } = context;
+    const { args: opponent } = request;
+
+    if (isEmpty(opponent)) {
+        send(i18n('fight.userisnotdefined'));
+
+        return request;
+    }
+    const fighters = ['you', opponent];
+    const winner = random(fighters);
+    const loser = random(fighters);
```

Comment:
чисто по логике получится мусор, тебе нужно выбрать вначале ОДНОГО, а потом противоположного... смотри, вначале положили в массив два id юзеров user и opponent, а потом перемешаем массив и результаты деструктуризации окажутся в случайном порядке 
```suggestion
    const [looser, winner] = shuffle(fighters);
```

`shuffle` это метод случайной сортировки из того же lodash